libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -40,3 +41,31 @@ void DPCTL_AsyncErrorHandler::operator()(
         }
     }
 }
+
+void error_handler(const std::exception &e,
+                   const char *file_name,
+                   const char *func_name,
+                   int line_num,
+                   int level)
+{
+    const char *verbose = std::getenv("DPCTL_VERBOSITY");
+    if (verbose) {
+        if (!strncmp(verbose, "error", 5))
```

Comment:
I think we should define `int requested_level = 0;`, then change it depending on the env variable. 

At the end we should do

```c++
   if (requested_level <= level) {
        std::cerr << e.what() << " in " << file_name << " at " << func_name
                        << " on line: " << line_num << std::endl;
   }
```